library/general/disallowanonymous/template.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sdisallowanonymous
+  annotations:
+    description: Disallows associating ClusterRole and Role resources to the system:anonymous user and system:unauthenticated group.
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sDisallowAnonymous
```

Comment:
It looks like this CT accepts parameters. We'll need a schema for that. Here's an example schema:

https://github.com/open-policy-agent/gatekeeper-library/blob/de1818ffdb03b2005dc8ef4beb23bfbadb85a63f/library/general/allowedrepos/template.yaml#L13-L22

Reply:
@maxsmythe Added a schema to CT.